index.bs

```diff
@@ -5854,6 +5884,14 @@ The following abstract operations operate on {{TransformStream}} instances at a
  1. Perform ! [$TransformStreamDefaultControllerClearAlgorithms$](|stream|.[=TransformStream/[[controller]]=]).
  1. Perform !
     [$WritableStreamDefaultControllerErrorIfNeeded$](|stream|.[=TransformStream/[[writable]]=].[=WritableStream/[[controller]]=], |e|).
+ 1. Perform ! [$TransformStreamUnblockWrite$](|stream|).
+</div>
+
+<div algorithm>
+ <dfn abstract-op lt="TransformStreamUnblockWrite"
```

Comment:
Nit-pick: these abstract ops are ordered alphabetically, so this new one should go after `TransformStreamSetBackpressure`.